### PR TITLE
API Products in Konnect

### PR DESCRIPTION
**Pull request by Guaris**
Changed naming of certain features/things to match new API Products naming (see linked DOCU ticket for detailed name substitutions)
Created a new section and docs for API Products
Revised the Getting Started pages to match the new workflow (and per Christians request)
Revised the Service Hub section to remove all pages but the Overview, which is now just a placeholder pointing people to the right places to go now and alerting them to the fact that Service Hub is getting updated.

To Do before release
- [x] comment out references/links to API spec; now tracking as post-GA deliverable. 

https://deploy-preview-5760--kongdocs.netlify.app/

### CODE REVIEW

File: app/konnect/org-management/teams-and-roles/roles-reference.md [hunk 1]
```diff
@@ -12,13 +12,13 @@ The following predefined roles are available in {{site.konnect_short_name}}:
...
-| Creator | Access to create new {{site.konnect_short_name}} services in Service Hub. The creator becomes the owner of the service they create, gaining admin access to the service. <br><br>This role does not provide access to creating sub-entities in a service such as service versions, implementations, API specs, or plugins. See the `Service Admin`, `Maintainer`, or `Plugins Admin` roles. |
-| Deployer | Access to implement and associate a {{site.konnect_short_name}} service version to a runtime group. <br><br> Must also have the `Deployer` role for the associated runtime group. |
-| Maintainer | Access to read, edit, and deploy a {{site.konnect_short_name}} service and its service versions, and manage its plugins. |
-| Plugins Admin | Access to install plugins on the {{site.konnect_short_name}} service versions and routes. <br><br> Must also have the `Admin` role in the associated runtime group. |
-| Publisher | Access to publish a {{site.konnect_short_name}} service to the Dev Portal. |
+| Admin | Admin of an existing {{site.konnect_short_name}} service. The admins have all write access related to a Gateway service and API product versions. |
```
Comment (sichvoge) [inline app/konnect/org-management/teams-and-roles/roles-reference.md:15]:
```suggestion
| Admin | Admin of an existing API product. The admins have all write access permissions related to a API product, API product version, etc. |
```

File: app/konnect/org-management/teams-and-roles/roles-reference.md [hunk 2]
```diff
@@ -12,13 +12,13 @@ The following predefined roles are available in {{site.konnect_short_name}}:
...
+| Application Registration | Access to enable or disable application registration for an API Product. |
+| Creator | Access to create new API product to productize services in API Products. The creator becomes the owner of the API product they create, gaining admin access to the API product. <br><br>This role does not provide access to creating sub-entities in an API product such as API product versions. See the `API Product Admin` or `Maintainer` roles. |
```
Comment (sichvoge) [inline app/konnect/org-management/teams-and-roles/roles-reference.md:17]:
```suggestion
| Creator | Access to create new API product in API Products. The creator becomes the owner of the API product they create, gaining admin access to the API product. This role does not provide access to creating sub-entities in an API product such as API product versions or API specs, or link the API product version to a Gateway service. See the Admin or Maintaine role. |
```

File: app/konnect/org-management/teams-and-roles/roles-reference.md [hunk 3]
```diff
@@ -12,13 +12,13 @@ The following predefined roles are available in {{site.konnect_short_name}}:
...
+| Deployer | Access to implement and associate a Gateway service to a runtime group. <br><br> Must also have the `Deployer` role for the associated runtime group. |
```
Comment (sichvoge) [inline app/konnect/org-management/teams-and-roles/roles-reference.md:18]: Please remove.

File: app/konnect/org-management/teams-and-roles/roles-reference.md [hunk 4]
```diff
@@ -12,13 +12,13 @@ The following predefined roles are available in {{site.konnect_short_name}}:
...
+| Maintainer | Access to read, edit, and deploy a Gateway service and its corresponding API product version versions, and manage its plugins. |
+| Plugins Admin | Access to install plugins on the Gateway service, runtime group, consumers, and routes. <br><br> Must also have the `Admin` role in the associated runtime group. |
```
Comment (sichvoge) [inline app/konnect/org-management/teams-and-roles/roles-reference.md:20]: Please remove.

File: app/konnect/org-management/teams-and-roles/roles-reference.md [hunk 5]
```diff
@@ -12,13 +12,13 @@ The following predefined roles are available in {{site.konnect_short_name}}:
...
-| Publisher | Access to publish a {{site.konnect_short_name}} service to the Dev Portal. |
+| Admin | Admin of an existing {{site.konnect_short_name}} service. The admins have all write access related to a Gateway service and API product versions. |
+| Application Registration | Access to enable or disable application registration for an API Product. |
+| Creator | Access to create new API product to productize services in API Products. The creator becomes the owner of the API product they create, gaining admin access to the API product. <br><br>This role does not provide access to creating sub-entities in an API product such as API product versions. See the `API Product Admin` or `Maintainer` roles. |
+| Deployer | Access to implement and associate a Gateway service to a runtime group. <br><br> Must also have the `Deployer` role for the associated runtime group. |
+| Maintainer | Access to read, edit, and deploy a Gateway service and its corresponding API product version versions, and manage its plugins. |
```
Comment (sichvoge) [inline app/konnect/org-management/teams-and-roles/roles-reference.md:19]:
```suggestion
| Maintainer | Access to fully manage an API product and its API product versions including app registration, publishing documentation, etc. |
```

File: app/konnect/org-management/teams-and-roles/roles-reference.md [hunk 6]
```diff
@@ -12,13 +12,13 @@ The following predefined roles are available in {{site.konnect_short_name}}:
...
+| Plugins Admin | Access to install plugins on the Gateway service, runtime group, consumers, and routes. <br><br> Must also have the `Admin` role in the associated runtime group. |
+| Publisher | Access to publish an API product to the Dev Portal. |
 | Viewer | Read-only access to all the configurations of a {{site.konnect_short_name}} service, including attributes, versions, Analytics reports, and plugins. |
```
Comment (sichvoge) [inline app/konnect/org-management/teams-and-roles/roles-reference.md:22]:
```suggestion
| Viewer | Read-only access on an API product including API product versions and its configuration, analytics, and documentation. |
```

**Review by sichvoge — CHANGES_REQUESTED** [review]
Few changes on the roles description.

**Comment by acgoldsmith** [comment]
@sichvoge - this is ready for release next week, and internal release today. 

Please note: 

Before merging the PR please publish the spec:
https://cloud.konghq.com/us/servicehub/d420333f-01b0-464e-a87a-97acc92c2026/overview

**Comment by sichvoge** [comment]
Please make the following changes on [this page](https://deploy-preview-5760--kongdocs.netlify.app/konnect/org-management/teams-and-roles/teams-reference/):

Both `Service Admin` and `Service Developer` become `API Product Admin` and `API Product Developer`. The descriptions changes also:

- `API Product Admin`: Allow creating and managing API products, including publishing API product versions to developer portal and enabling application registration.
- `API Product Develope`r: Allow creating and managing versions of API products.

**Review by sichvoge — APPROVED** [review]
LGTM 👍

**Pull request merged**